FatmaQunnies/app/route.py

```diff
@@ -0,0 +1,74 @@
+from FatmaQunnies.classes import urgent_task
+from fastapi import FastAPI
+import uuid
+from classes import *
+from classes.model import TaskModel ,TaskUpdate
+
+
+app = FastAPI()
+tasks = []
```

Comment:
Why use a list as a database when you can use a dictionary with the task ID as the key and the task as the value?

Reply:
Yes I noticed I used it in put and I forgot to make a change for the rest